src/components/WorkersList/WorkersList.tsx

```diff
@@ -0,0 +1,36 @@
+import { WorkerItem } from "../WorkerItem/WorkerItem";
+import { Worker } from "../../moudels/Worker";
+import { WorkersContext } from "../WorkersProvider/WorkersProvider";
+import { useContext } from "react";
+import { fetchWorkersList } from "../../services/Workers";
+import { fetchLocalTime } from "../../services/LocalTime";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+export const WorkersList = () => {
+  const { workersList, setWorkersList } = useContext(WorkersContext);
+  if (workersList.length == 0) {
```

Comment:
All the requests are being sent simultaneously which is great, but some are sent twice - for example, if there are 2 guys living in the same place, don't send 2 requests for city and 2 requests for timezone. Send as little as possible.